db/migrate/20150926131928_add_attachment_image_to_ex_links.rb

```diff
@@ -0,0 +1,11 @@
+class AddAttachmentImageToExLinks < ActiveRecord::Migration
+  def self.up
+    change_table :ex_links do |t|
+      t.attachment :image
```

Comment:
Change this to `t.string :image` to work with CarrierWave (if you want to change).